guides/source/documents.yaml

```diff
@@ -140,6 +140,11 @@
       work_in_progress: true
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Active Record and PostgreSQL
+      work_in_progress: true
+      url: active_record_postgresql.html
+      description: This guide covers PostgreSQL specific usage of Active Record.
```

Comment:
I'm :-1: to add a document to the list that is not in the quality that we want, even as Work in progress. @fxn WDYT?